homeassistant/components/solarlog/config_flow.py

```diff
@@ -36,7 +36,7 @@ def __init__(self) -> None:
         """Initialize the config flow."""
         self._errors: dict = {}
 
-    def _host_in_configuration_exists(self, host) -> bool:
+    def _host_in_configuration_exists(self, host: str) -> bool:
```

Comment:
This can be removed by the way, `self._async_abort_entries_match` is what this is doing

Reply:
But for followup